__tests__/app.test.js

```diff
@@ -102,13 +101,59 @@ describe("GET/api/articles", () => {
         });
       });
   });
-  test("articles in the array should be sorted by date in descending order", () =>{
+  test("articles in the array should be sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((res) => {
+        const articles = res.body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+});
+describe("GET/api/articles/:article_id/comments", () => {
+  test("should respond with an array of comments for given article_id, each with the correct properties", () => {
     return request(app)
-    .get("/api/articles")
-    .expect(200)
-    .then((res) => {
-      const articles = res.body;
-      expect(articles).toBeSortedBy('created_at', {descending: true})
-    })
-  })
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        expect(res.body).toHaveLength(11);
+        res.body.forEach((comment) => {
+          expect(comment).toMatchObject({
+            body: expect.any(String),
+            votes: expect.any(Number),
+            author: expect.any(String),
+            article_id: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
+  test("comments in the array should be sorted by date in descending order, the most recent comment first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((res) => {
+        const comments = res.body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+  describe("error handling for GET comments", () => {
+    test("status:400, responds with an error message when passed a bad article ID", () => {
+      return request(app)
+        .get("/api/articles/notAnID/comments")
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
+    });
+    test("responds with status 200 and returns an empty array when given a valid article ID when article does not have comments", () => {
```

Comment:
This is good, but we should have a separate test and some feedback for if the article just doesn't exist. In both the case above, and the case where the article doesn't exist, we will get an empty array in response from the db. We need to be sure we're feeding back the right message to the client.